x-pack/plugins/aiops/public/components/log_categorization/sampling_menu/random_sampler.ts

```diff
@@ -28,27 +28,33 @@ export type RandomSamplerProbability = number | null;
 
 export const RANDOM_SAMPLER_SELECT_OPTIONS: Array<{
   value: RandomSamplerOption;
-  text: string;
+  inputDisplay: string;
   'data-test-subj': string;
 }> = [
   {
     'data-test-subj': 'aiopsRandomSamplerOptionOnAutomatic',
     value: RANDOM_SAMPLER_OPTION.ON_AUTOMATIC,
-    text: i18n.translate('xpack.aiops.logCategorization.randomSamplerPreference.onAutomaticLabel', {
-      defaultMessage: 'On - automatic',
-    }),
+    inputDisplay: i18n.translate(
+      'xpack.aiops.logCategorization.randomSamplerPreference.onAutomaticLabel',
+      {
+        defaultMessage: 'On - automatic',
```

Comment:
Guess it isn't related to the changes here, but sometimes can see too high precision for the probability:

<img width="442" alt="Screenshot 2024-05-14 at 17 29 15" src="https://github.com/elastic/kibana/assets/7405507/ad282167-b5fd-460f-8be1-ec19f85c5f9a">

Reply:
Fixed in https://github.com/elastic/kibana/pull/178916/commits/3ea27fc97da1f99cf63ed5168f19e1b57cf9f8c1